libs/tour/src/lib/services/tour-service/tour.service.ts

```diff
@@ -417,8 +422,11 @@ export class NgxTourService implements OnDestroy {
 		this.previousStepSubject.next(this.currentStepSubject.getValue());
 		this.currentStepSubject.next(currentStep);
 
-		// Iben: Scroll item into view if it's not in view
 		this.runInBrowser(() => {
+			// Iben: Scroll to the top before each step to get consistent behavior when going back and forth
+			window.scrollTo({ top: 0 });
```

Comment:
Why is this needed? Which issue does this solve? If stap 1 and 2 are both outside the current viewport, the user will

1. Scroll down into the step
2. Scroll to the top
3. Scroll into the second step

They will be rubber banded with each step

Reply:
Given that the scroll to top goes instantly, you do not get a rubber-banded feeling at al. It wouldn't have been added if it gave that visual effect 😅 

As mentioned in the comment, if you don't do this, you get an inconsistent behaviour. Here's an example.

1. At the start, the item is in view. We're not scrolling to the element, as we don't have to.
![afbeelding](https://github.com/studiohyperdrive/ngx-tools/assets/10544531/cdc45d2f-73f9-4791-a3e9-dbee7a342b4e)

2. We continue through the tour, and navigate to an element that's lower, so we scroll to that element. Now the above element is out of the viewport. Which isn't an issue.

3. Instead of continuing now, we however go back. But now, the item is not in the viewport and we scroll to it. The browser considers it to be in view, but now you get a completely different view than you had initially.
![afbeelding](https://github.com/studiohyperdrive/ngx-tools/assets/10544531/b594e4c0-f24a-47d7-990d-9ca4717f217b)

If you don't scroll back to the top before each step, you get inconsistent views between going back and forth, which is not something we want. We want the effect to be the same regardless of whether we're going back or forward.

Reply:
As long as the user does not notice, I am a-ok with this implementation! 🙂